tests/test_helpers.py

```diff
@@ -264,16 +264,21 @@ def test_parse_date_from_string_invalid_date_returns_none():
 
 
 def test_validate_date_success():
-    assert validate_date("1930 ", "1234") is True
+    assert validate_date("1930", "1234") is True
 
 
 def test_validate_date_invalid_date_logs_error(caplog):
     assert validate_date("circa 1930s", "1234") is False
     assert (
-        "Record # '1234' has a date that couldn't be parsed: circa 1930s"
+        "Record # '1234' has a date that couldn't be parsed: 'circa 1930s'"
     ) in caplog.text
 
 
+def test_validate_date_whitespace_date_logs_error(caplog):
+    assert validate_date("30  ", "1234") is False
+    assert ("Record # '1234' has a date that couldn't be parsed: '30  '") in caplog.text
```

Comment:
Not directly related to this line, but are we sure OpenSearch is treating "30" as a date the same way it might be meant in MARC? Adding the strip() prevents the problem... but are we ending up with data that matches the intent of the cataloger? I'm not saying we aren't, but since there is no test here to clarify these types of edge cases and whether we want "30" to be valid I'm not sure if we are doing the correct thing by just stripping and not confirming the intent remains for these yet.

Reply:
Do you have a record id we can compare in OpenSearch with the record in Alma to ensure the two systems are treating two digit dates the same way?